biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -139,37 +206,30 @@ export const SimulatePage = () => {
           <div className="simboard-container">
             <h2>Simulation Board</h2>
             <div className="simboard-controls">
-              <button onClick={() => playSimulation()}>Play</button>
-              <button onClick={() => setCurrentStateFrame(0)}>Reset</button>
-            </div>
-            <div>
-              Time of day:{' '}
-              {currentStateFrame % 12 === 0 ? 12 : currentStateFrame % 12}:00{' '}
-              {currentStateFrame >= 12 ? 'PM' : 'AM'}
+              <button
+                onClick={() => {
+                  const pauseFn = play();
+                  setControlSimultation({ pauseFN: pauseFn });
+                }}
+              >
+                Play
+              </button>
+              <button onClick={() => pauseSimulation()}>Pause</button>
```

Comment:
another thing I noticed, it seems like when I click "pause", it moves one more state, and then pauses. It should pause on the current state